test/Helper/HeadLinkTest.php

```diff
@@ -319,18 +321,24 @@ public function testConditionalStylesheetCreationNoIEWidthSpaces()
         $this->assertStringContainsString('<!--<![endif]-->', $string);
     }
 
-    public function testSettingAlternateWithTooFewArgsRaisesException()
+    public function argumentCountProvider() : iterable
     {
-        try {
-            $this->helper->setAlternate('foo');
-            $this->fail('Setting alternate with fewer than 3 args should raise exception');
-        } catch (ViewException $e) {
-        }
-        try {
-            $this->helper->setAlternate('foo', 'bar');
-            $this->fail('Setting alternate with fewer than 3 args should raise exception');
-        } catch (ViewException $e) {
-        }
+        return [
+            'One' => [1],
+            'Two' => [2],
+        ];
+    }
+
+    /** @dataProvider argumentCountProvider */
```

Comment:
Please run the coding standard check. I don't know if the short DocBlock is allowed for methods.

Reply:
Just checked this with coding-std ~2 and the short doc block is preferred. ~1 doesn't complain.